sublime-security/sublime-platform
# More updates to setup script experience

How to test (s/o @hugh for this):

`curl -sL https://raw.githubusercontent.com/sublime-security/sublime-platform/josh.setup_script/install-and-launch.sh | remote_branch=josh.setup_script bash`

Note that Github caches the raw content and [only updates every `5 minutes](https://stackoverflow.com/questions/46551413/github-not-update-raw-after-commit). There doesn't seem to be a way to bypass this, except possibly by requesting the commit itself instead of the branch.

Local testing: `clone_platform=false bash install-and-launch.sh`

<img width="1113" alt="image" src="https://user-images.githubusercontent.com/11003450/213937669-1c39186f-8f83-475e-ad51-4db16af51faf.png">

<img width="860" alt="image" src="https://user-images.githubusercontent.com/11003450/213937656-ffd50811-0044-4d3c-aa26-098ae046e07e.png">
